core/storage/suggestion/gae_models.py

```diff
@@ -1990,3 +2000,1275 @@ def export_data(
+                base_models.EXPORT_POLICY.EXPORTED,
+            'accepted_questions_without_reviewer_edits_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'rejected_questions_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'first_contribution_date':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'last_contribution_date':
+                base_models.EXPORT_POLICY.EXPORTED
+        })
+
+    @classmethod
+    def apply_deletion_policy(cls, user_id: str) -> None:
+        """Delete instances of QuestionSubmitterTotalContributionStatsModel
+        for the user.
+
+        Args:
+            user_id: str. The ID of the user whose data should be deleted.
+        """
+        datastore_services.delete_multi(
+            cls.query(cls.contributor_id == user_id).fetch(keys_only=True))
+
+    @classmethod
+    def export_data(
+        cls, user_id: str
+    ) -> Dict[str, Dict[str, Union[str, int, List[str]]]]:
+        """Exports the data from QuestionSubmitterTotalContributionStatsModel
+        into dict format for Takeout.
+
+        Args:
+            user_id: str. The ID of the user whose data should be exported.
+
+        Returns:
+            dict. Dictionary of the data from
+            QuestionSubmitterTotalContributionStatsModel.
+        """
+        user_data = {}
+        stats_models: Sequence[QuestionSubmitterTotalContributionStatsModel] = ( # pylint: disable=line-too-long
+            cls.get_all().filter(cls.contributor_id == user_id).fetch())
+        for model in stats_models:
+            user_data = {
+                'topic_ids_with_question_submissions': (
+                    model.topic_ids_with_question_submissions),
+                'recent_review_outcomes': (
+                    model.recent_review_outcomes),
+                'recent_performance': (
+                    model.recent_performance),
+                'overall_accuracy': (
+                    model.overall_accuracy),
+                'submitted_questions_count': (
+                    model.submitted_questions_count),
+                'accepted_questions_count': (
+                    model.accepted_questions_count),
+                'accepted_questions_without_reviewer_edits_count': (
+                    model.accepted_questions_without_reviewer_edits_count),
+                'rejected_questions_count': (
+                    model.rejected_questions_count),
+                'first_contribution_date': (
+                    model.first_contribution_date.isoformat()),
+                'last_contribution_date': (
+                    model.last_contribution_date.isoformat())
+            }
+        return user_data
+
+
+class QuestionReviewerTotalContributionStatsModel(base_models.BaseModel):
+    """Records the question review stats for contributor admin dashboard.
+    There is one instance of this model per contributor_id.
+    Its IDs is same as the user_id of the contributor.
```

Comment:
Same comments as above, throughout this class.

Reply:
Updated best upto my knowledge. PTAL. Thanks